client/Components/GameBoard/IdentityCard.jsx

```diff
@@ -7,8 +7,8 @@ import QRCode from 'qrcode';
 import { withTranslation } from 'react-i18next';
 
 class IdentityCard extends React.Component {
-    constructor(props) {
```

Comment:
It is 'proper' to pass the props to the base class, so not sure why you're removing it :)

Reply:
Googles "props base class"
 realization Dawns
I watch the world burn.